### DISCUSSION TITLE
error in tap CI: brew link m4

### DISCUSSION THREAD

**Discussion by dpo** [Homebrew/discussions#2927] [Tap maintenance and Homebrew development]
Hi. I just set up the defaults yml scripts for CI with GitHub Actions to build bottles for a tap. I was able to update and build bottles for all formulae.

However, upon updating one of the formulae in a subsequent PR, CI fails with a `brew link m4` error that I'm not understanding:

https://github.com/coin-or-tools/homebrew-coinor/runs/5073713270?check_suite_focus=true#step:8:67

All dependents appear to be correctly installing from bottles. `m4` is not a direct dependency of any formula in the tap.

Any ideas what could be causing that error?

Many thanks in advance!

**Comment by carlocab** [comment]
What's the output of
```
brew deps --tree --include-test coin-or-tools/coinor/symphony
```
?

**Comment by dpo** [comment]
https://gist.github.com/dpo/ac4942aa0e9c3d0b8ad2df0e1c5d8b26

**Reply by carlocab** [reply]
Yea that's a bug. Probably here:

https://github.com/Homebrew/homebrew-test-bot/blob/e486c73e3067672689ab9d0c1c50fe2a2d7e6c62/lib/tests/formulae_dependents.rb#L213-L231

You do have `m4` as a (recursive) build dep, but that shouldn't matter. Could you file an issue at homebrew/homebrew-test-bot?

**Reply by dpo** [reply]
Done. Many thanks!
https://github.com/Homebrew/homebrew-test-bot/issues/783
